config/local_docker/genesis_config.json

```diff
@@ -13,17 +13,11 @@
     "l1FeeOverhead": "0x1",
     "l1FeeScalar": "0x1",
     "alloc": {
-        "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266": {
-            "balance": "10000000000000000000"
-        },
-        "0x70997970c51812dc3a010c7d01b50e0d17dc79c8": {
-            "balance": "10000000000000000000"
-        },
         "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc": {
```

Comment:
should we use `--alloc` flag for genesis generation? will these account change each time when generating secrets? cc @ujvl

Reply:
🤷